tests/integration_tests/data_steward/tools/top_heel_errors_test.py

```diff
@@ -100,7 +100,10 @@ def test_top_heel_errors_no_hpo_prefix(self):
         actual_results = comparison_view(dataset_errors)
         self.assertCountEqual(actual_results, expected_results)
 
-    def test_top_heel_errors_all_hpo(self):
+    @mock.patch('tools.top_heel_errors.get_hpo_ids')
+    def test_top_heel_errors_all_hpo(self, mock_hpo_ids):
+        hpo_ids = [HPO_NYC, HPO_PITT]
+        mock_hpo_ids.return_value = hpo_ids
```

Comment:
I was wondering why you changed the list comprehension.  I understand now.  This looks good.